apps/omg_api/lib/state/transaction.ex

```diff
@@ -26,11 +26,12 @@ defmodule OMG.API.State.Transaction do
   @max_inputs 4
   @max_outputs 4
 
-  defstruct [:inputs, :outputs]
+  defstruct [:inputs, :outputs, metadata: ""]
```

Comment:
Since this field is binary (not printable in general case) use `<<>>` instead of `""`.